test/lsqb_test/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_kuzu_test(lsqb_test lsqb_test.cpp)
```

Comment:
I think we can just simply call this test as `lsqb` without the `_test` suffix. Also the directory name.